docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -138,7 +367,54 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use atparse-formatted template file ``atparse.txt`` with content
+
+.. code:: sh
+  @[greeting], @[recipient]!
+
+* Convert an atparse-formatted template file to Jinja2 format:
+
+  .. code:: sh
+
+    $ uw config translate --input-file atparse.txt --input-format atparse --output-format jinja2
+    {{greeting}}, {{recipient}}!
+
+  Shell redirection via ``|``, ``>``, et al may also be used to stream output to a file, another process, etc.
+
+* Convert the template to a file via command-line argument:
+
+  .. code:: sh
+
+    $ uw config realize --input-file values.yaml --output-file realized.yaml
+
+  The content of ``jinja2.txt``:
+
+  .. code:: sh
+
+    {{greeting}}, {{recipient}}!
+
+* With the ``--dry-run`` flag specified, nothing is written to ``stdout`` (or to a file if ``--output-file`` is specified), but a report of what would have been written is logged to ``stderr``:
+
+  .. code:: sh
+
+    $ uw config translate --input-file atparse.txt --input-format atparse --output-format jinja2 --dry-run
+    [2024-01-03T16:41:13]     INFO {{greeting}}, {{recipient}}!
+
+
+* If an input file is read alone from ``stdin``, ``uw`` know how to parse its content as we must always specify the formats:
+
+  .. code:: sh
+
+    $ cat atparse.txt | uw config translate --input-format atparse --output-format jinja2
+    {{greeting}}, {{recipient}}!
+
+
+* Request verbose log output:
+
+  .. code:: sh
+
+    $ uw config translate --input-file atparse.txt --input-format atparse --output-format jinja2 --verbose
+    {{greeting}}, {{recipient}}!
```

Comment:
Do we really not get any log lines with `--verbose`? If not, IMO we might as well remove this bullet, as it doesn't really demonstrate anything.

Reply:
Yeah, or with validate. Both should probably either add more info or remove quiet/verbose. The `--quiet` isn't mentioned here or there because its even more useless